ert_gui/simulation/run_dialog.py

```diff
@@ -302,6 +310,57 @@ def run():
         worker_thread.started.connect(worker.consume_and_emit)
         self._worker_thread.start()
 
+    def startSimulationErt3(self, ensemble_evaluator):
```

Comment:
should we add typing to new functions like this? (easening future typing of everything)

Reply:
Now it's added